docs/api/rest.md

```diff
@@ -488,7 +488,12 @@ curl -X GET -H "Authorization: Bearer ABCDEFGH" \
 
 Returns the list of calendar events for the specified calendar entity_id between the `start` and `end` times (exclusive).
 
-The events in the response have a `start` and `end` that contain either `dateTime` or `date` for an all day event.
+The events in the response have a:
```

Comment:
I think we should revert this part -- i just left in the start/end description because it may not be obvious but I think the summary, description, location schema is self explanatory from the example.

How about linking to https://developers.home-assistant.io/docs/core/entity/calendar#calendarevent here for details on the extended attributes?

Reply:
Did both